File: jwtprac/src/main/java/com/member/jwtprac/entity/UserEntity.java
package com.member.jwtprac.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;

@Entity
@Data
public class UserEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;

    private String username;

    private String password;

    private String role;
}

File: jwtKotlin/src/main/kotlin/com/member/jwtkotlin/entity/UserEntity.kt
package com.member.jwtkotlin.entity

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id

@Entity
class UserEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Int = 0

    var username: String? = null
    var password: String? = null
    var nickname: String? = null
    var role: String? = null
}